src/components/ProfileHeader/ProfileHeader.module.css

```
@@ -0,0 +1,64 @@
+.ProfileHeader {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  background-color: white;
+  border-radius: 4px;
+  border: 1px solid silver;
+  text-align: center;
+}
+
+.Avatar {
+  max-width: 200px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 25px;
+}
+
+.Text {
+  text-align: center;
+  align-self: stretch;
+}
+
+.Center {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.FilledLike {
+  text-align: center;
+}
+
+.Comment {
```

let's make it flex and center the items
```suggestion
.FilledLike {
  display: flex;
  align-items: center;
```